[2024-11-25T09:00:20.164+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-25T09:00:20.224+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_PipelineV31.fetchData scheduled__2024-11-25T08:00:00+00:00 [queued]>
[2024-11-25T09:00:20.249+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_PipelineV31.fetchData scheduled__2024-11-25T08:00:00+00:00 [queued]>
[2024-11-25T09:00:20.250+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 4
[2024-11-25T09:00:20.287+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): fetchData> on 2024-11-25 08:00:00+00:00
[2024-11-25T09:00:20.332+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_PipelineV31', 'fetchData', 'scheduled__2024-11-25T08:00:00+00:00', '--job-id', '31', '--raw', '--subdir', 'DAGS_FOLDER/etlDag.py', '--cfg-path', '/tmp/tmpobqzwlwr']
[2024-11-25T09:00:20.338+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=4693) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-11-25T09:00:20.342+0000] {standard_task_runner.py:105} INFO - Job 31: Subtask fetchData
[2024-11-25T09:00:20.342+0000] {standard_task_runner.py:72} INFO - Started process 4722 to run task
[2024-11-25T09:00:20.695+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_PipelineV31.fetchData scheduled__2024-11-25T08:00:00+00:00 [running]> on host 7939e06e2b4b
[2024-11-25T09:00:21.671+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Oshith Roshantha' AIRFLOW_CTX_DAG_ID='etl_PipelineV31' AIRFLOW_CTX_TASK_ID='fetchData' AIRFLOW_CTX_EXECUTION_DATE='2024-11-25T08:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-25T08:00:00+00:00'
[2024-11-25T09:00:21.695+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-25T09:00:23.510+0000] {_universal.py:506} INFO - Request URL: 'https://eqdatastorage.blob.core.windows.net/staging-data/temp-json'
Request method: 'PUT'
Request headers:
    'Content-Length': '315'
    'x-ms-blob-type': 'REDACTED'
    'x-ms-version': 'REDACTED'
    'Content-Type': 'application/octet-stream'
    'Accept': 'application/xml'
    'User-Agent': 'azsdk-python-storage-blob/12.24.0 Python/3.12.7 (Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36)'
    'x-ms-date': 'REDACTED'
    'x-ms-client-request-id': 'b49f0ae0-ab0b-11ef-9f8e-0242ac120003'
    'Authorization': 'REDACTED'
A body is sent with the request
[2024-11-25T09:00:24.638+0000] {_universal.py:545} INFO - Response status: 201
Response headers:
    'Content-Length': '0'
    'Content-MD5': 'REDACTED'
    'Last-Modified': 'Mon, 25 Nov 2024 09:00:25 GMT'
    'ETag': '"0x8DD0D2F9A03A89C"'
    'Server': 'Windows-Azure-Blob/1.0 Microsoft-HTTPAPI/2.0'
    'x-ms-request-id': 'c849ec51-101e-00f2-7818-3f8003000000'
    'x-ms-client-request-id': 'b49f0ae0-ab0b-11ef-9f8e-0242ac120003'
    'x-ms-version': 'REDACTED'
    'x-ms-content-crc64': 'REDACTED'
    'x-ms-request-server-encrypted': 'REDACTED'
    'Date': 'Mon, 25 Nov 2024 09:00:24 GMT'
[2024-11-25T09:00:24.646+0000] {python.py:240} INFO - Done. Returned value was: {'type': 'FeatureCollection', 'metadata': {'generated': 1732525224000, 'url': 'https://earthquake.usgs.gov/fdsnws/event/1/query?format=geojson&starttime=2024-11-25T07%3A00%3A00&endtime=2024-11-25T08%3A00%3A00&minmagnitude=4', 'title': 'USGS Earthquakes', 'status': 200, 'api': '1.14.1', 'count': 0}, 'features': []}
[2024-11-25T09:00:24.726+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-25T09:00:24.728+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_PipelineV31, task_id=fetchData, run_id=scheduled__2024-11-25T08:00:00+00:00, execution_date=20241125T080000, start_date=20241125T090020, end_date=20241125T090024
[2024-11-25T09:00:24.861+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-25T09:00:25.211+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-11-25T09:00:25.219+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
